run_cumulus_task.py

```diff
@@ -26,5 +32,10 @@ def run_cumulus_task(task_function, cumulus_message, context):
     full_event = adapter.loadRemoteEvent(cumulus_message)
     nested_event = adapter.loadNestedEvent(full_event, vars(context))
     message_config = nested_event.get('messageConfig', {})
-    task_response = task_function(nested_event, context)
+
+    try:
+        task_response = task_function(nested_event, context)
+    except:
+        raise Exception("WorkflowError")
```

Comment:
It's a small thing, but the Java and JS implementations check to see that `os.environ.get('CUMULUS_MESSAGE_ADAPTER_DISABLED') == 'true'`.